www/layouts/home.html

```diff
@@ -151,10 +151,10 @@ <h2>Your Donation Makes a Difference</h2>
       <h2 class="text-uppercase text-center">Our Corporate and Foundation Supporters</h2>
       <div class="row w-100 justify-content-center">
         <div class="col-12 col-md-3 col-xl-2 d-flex align-items-center justify-content-center py-3">
```

Comment:
```suggestion
        <div class="col-12 col-md-3 col-xl-2 d-flex justify-content-center py-md-2 py-3">
```
Improves this alignement:
<img width="926" alt="Screenshot 2022-04-13 at 2 21 35 PM" src="https://user-images.githubusercontent.com/93309234/163145331-37f75e16-68df-4d5a-b6f4-d8ade737cd5c.png">
(note: This change is only for `ab_initio_logo` logo div)